package-lock.json

```diff
@@ -23,7 +23,6 @@
         "ajv": "^8.12.0",
         "ajv-draft-04": "^1.0.0",
         "ajv-formats": "^2.1.1",
-        "aws-sdk": "^2.1354.0",
```

Comment:
Any reason we are removing `aws-sdk` dependency in this PR ?

Reply:
I haven't removed it, may happened due to latest pull from base branch